PWT/Fantina/Tournament3.yaml

```diff
@@ -0,0 +1,60 @@
+---
+species: Giratina
+setname: PWT-Fantina Tournament3
+tags: [in-game, PWT-Fantina Tournament3, trainer+fantina]
+item: [Ghost Gem]
```

Comment:
another gem

Reply:
Might scrap Tournament3 set until we can add Gems in then

Reply:
Could you replace with Shadow Plate or Spell Tag? Those are the closest equivalents.

Reply:
I would if PWT wasn't trying to be as accurate as possible to the source games.